src/pages/settings-general/settings-general.spec.ts

```diff
@@ -41,4 +42,28 @@ describe("SettingsGeneralPage", () => {
   });
 
   it("should create component", () => expect(comp).toBeDefined())
+
```

Comment:
Small note about feature branches. Try and keep the work you do in a feature branch to the point. Don't go adding things which are not part of the feature you are working on. The below unit tests (and other changes in the pull request) are not part of the nav header feature request. 

By keeping these pull requests to the point you make it easier for other developers to review them.